src/main/java/seedu/simplykitchen/model/util/SampleDataUtil.java

```diff
@@ -19,17 +20,17 @@
     public static Food[] getSampleFoods() {
         return new Food[] {
             new Food(new Description("Apple Pie"), new Priority("high"), new ExpiryDate("30-9-2020"),
-                    getTagSet("Frozen")),
+                    new Quantity("1 pie"), getTagSet("Frozen")),
             new Food(new Description("Bread"), new Priority("medium"), new ExpiryDate("1-10-2020"),
-                    getTagSet("Wholemeal", "400g")),
+                    new Quantity("2 loaf"), getTagSet("Wholemeal", "400g")),
             new Food(new Description("Chocolate Cake"), new Priority("high"), new ExpiryDate("1-1-2021"),
-                    getTagSet("$50")),
+                    new Quantity("0.7 lb"), getTagSet("$50")),
             new Food(new Description("Donuts"), new Priority("low"), new ExpiryDate("1-11-2021"),
-                    getTagSet("Plain")),
+                    new Quantity("5 donuts"), getTagSet("Plain")),
```

Comment:
Is using the singular verb "donut" be better? What if the same user add another food item of 1 donut?